lib/providers/my_list_section_updates.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:brunstadtv_app/graphql/client.dart';
+import 'package:brunstadtv_app/graphql/queries/sections.graphql.dart';
+import 'package:riverpod/riverpod.dart';
+
+import '../graphql/queries/page.graphql.dart';
+
+class _MyListUpdateNotifier extends StateNotifier<List> {
+  _MyListUpdateNotifier() : super([]);
+  void notify() => state = [];
+}
+
+final myListUpdateNotifierProvider = StateNotifierProvider<_MyListUpdateNotifier, List>((ref) {
+  return _MyListUpdateNotifier();
+});
+
+final myListSectionUpdatesProvider = FutureProvider.family<Fragment$Section?, String>((ref, sectionId) async {
```

Comment:
Idea:
- send an event on `globalEventBus` called MyListChangedEvent(); which might not need any data
- in SectionUpdatesHandler, if it's a my-list section, listen to the eventbus filtered by MyListChangedEvent, and refetch the section directly in that widget.

Reply:
That's a good idea.